media/js/firefox/australis/growth-firstrun-test2.js

```diff
@@ -41,7 +41,8 @@ function onYouTubeIframeAPIReady() {
             Mozilla.UITour.showFirefoxAccounts();
         };
 
-        gaTrack(['_trackEvent', pageId + ' Page Interactions - New Firefox Tour', 'button click', 'Get Started with Sync'], goToAccounts);
```

Comment:
Since `pageId` is removed here, we can remove the declaration on line 8 as well.